templates/perf-infra/manifests.yaml

```diff
@@ -0,0 +1,209 @@
+# Stress test to test upper bounds of concurrent pods
```

Comment:
My only change is that everything should go into the manifest. Including any docs.